omnibus/cookbooks/omnibus-supermarket/attributes/default.rb

```diff
@@ -72,6 +72,7 @@
 default['supermarket']['install_directory'] = '/opt/supermarket'
 default['supermarket']['app_directory'] = "#{node['supermarket']['install_directory']}/embedded/service/supermarket"
 default['supermarket']['log_directory'] = '/var/log/supermarket'
+default['supermarket']['log_directory']['mode'] = '0700'
```

Comment:
Sadly, we won't be able to set a `['supermarket']['log_directory']['mode']` subkey when the parent key `['supermarket']['log_directory']` has a value type of `String`.